metadata/tests/unit/proxy/test_neo4j_proxy.py

```diff
@@ -53,19 +53,88 @@ def setUp(self) -> None:
+                                               'data_type': 'struct<c1:string,c2:array<string>,'
+                                                            'c3:map<string,string>,c4:struct<c5:int,c6:int>>'}},
+                                     {'node': {'kind': 'scalar', 'name': 'c1',
+                                               'key': 'hive://gold.foo_schema/foo_table/bar_id_1/type/bar_id_1/_inner_'
+                                                      '/c1',
+                                               'data_type': 'string', 'sort_order': 0}},
+                                     {'node': {'kind': 'array', 'name': 'c2',
+                                               'key': 'hive://gold.foo_schema/foo_table/bar_id_1/type/bar_id_1/_inner_'
+                                                      '/c2',
+                                               'data_type': 'array<string>', 'sort_order': 1}},
+                                     {'node': {'kind': 'map', 'name': 'c3',
+                                               'key': 'hive://gold.foo_schema/foo_table/bar_id_1/type/bar_id_1/_inner_'
+                                                      '/c3',
+                                               'data_type': 'map<string,string>', 'sort_order': 2}},
+                                     {'node': {'kind': 'struct', 'name': 'c4',
+                                               'key': 'hive://gold.foo_schema/foo_table/bar_id_1/type/bar_id_1/_inner_'
+                                                      '/c4',
+                                               'data_type': 'struct<c5:int,c6:int>', 'sort_order': 3}},
+                                     {'node': {'kind': 'scalar', 'name': 'map_key',
+                                               'key': 'hive://gold.foo_schema/foo_table/bar_id_1/type/bar_id_1/_inner_'
+                                                      '/c3/map_key',
+                                               'data_type': 'string'}},
+                                     {'node': {'kind': 'scalar', 'name': 'map_value',
+                                               'key': 'hive://gold.foo_schema/foo_table/bar_id_1/type/bar_id_1/_inner_'
+                                                      '/c3/map_value',
+                                               'data_type': 'string'}},
+                                     {'node': {'kind': 'scalar', 'name': 'c5',
+                                               'key': 'hive://gold.foo_schema/foo_table/bar_id_1/type/bar_id_1/_inner_'
+                                                      '/c4/c5',
+                                               'data_type': 'int', 'sort_order': 0}},
+                                     {'node': {'kind': 'scalar', 'name': 'c6',
+                                               'key': 'hive://gold.foo_schema/foo_table/bar_id_1/type/bar_id_1/_inner_'
```

Comment:
we have a problem, I realized: `type` is a pretty frequent column name! It's even a possible table name. Can we change the databuilder to make it `column//type/the_type_path`. The double slash will disambiguate.